common/dependabot-common.gemspec

```diff
@@ -7,13 +7,18 @@ Gem::Specification.new do |spec|
   spec.version      = Dependabot::VERSION
   spec.summary      = "Shared code used between Dependabot package managers"
   spec.description  = "Automated dependency management for Ruby, JavaScript, " \
-                      "Python, PHP, Elixir, Rust, Java, .NET, Elm and Go"
+                      "Python, Docker, PHP, Elixir, Rust, Java, .NET, Elm and Go"
```

Comment:
Dart too?

Reply:
Also, can we maybe sort this list?

Reply:
oof, I didn't mean to ship this with this PR, as I've got another I'm working on right now to rewrite these descriptions... they're super confusion on RubyGems (that was actually what prompted me wanting to include fixing this as part of our internal OKR around contributor docs)

Reply:
Removed from this PR.